Source/Core/Core/HW/GCMemcard/GCMemcardRaw.cpp

```diff
@@ -92,6 +93,12 @@ MemoryCard::~MemoryCard()
   }
 }
 
+void MemoryCard::FlushWrites()
+{
+  m_flush_trigger.Set();
```

Comment:
Won't this line cause the flush thread to exit after flushing? That's not what we want.

Reply:
Oh.. Yeah.. I misread what that was being used for..

Reply:
Memory cards are no longer addressed in this PR.